src/main/java/io/jsonwebtoken/BadAudienceJwtException.java

```diff
@@ -0,0 +1,11 @@
+package io.jsonwebtoken;
+
+public class BadAudienceJwtException extends ClaimJwtException {
```

Comment:
Nice! Right now there's no distinction between missing and unexpected (which I like better than `incorrect` as it matches with `expect` from the parser)